Backpack/StarRating/Classes/BPKStarRating.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+#import "BPKStar.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(StarRating) @interface BPKStarRating : UIView
+
+/**
+ * Rating value the star rating view will displaying by rounding the value to full and half stars between 0.0 and 5.0.
+ * Set this property to render new rating.
+ * The default value is 0.0, in case of setting a value smaller than 0.0 or greater than 5.0 the view will show 0.0 or 5.0.
+ */
+@property (nonatomic) float rating;
```

Comment:
What happens with a value like `4.3`? Can we include this in the doc string?

Reply:
Done